content/reference-rippled.md

```diff
@@ -2190,17 +2192,18 @@ The request includes the following parameters:
+| `ledger_index_max` | Integer                                    | Use to specify the most recent ledger to include transactions from. A value of `-1` instructs the server to use the most recent validated ledger version available. This field is optional if you use `ledger_index_min`, however you can use these fields in tandem to define a range of ledgers. This field is optional and unnecessary if you use `ledger_hash` or `ledger_index`. |
+| `ledger_hash`      | String                                     | Use to look for transactions from a single ledger only. This field is optional and unnecessary if you use `ledger_index_min`, `ledger_index_max`, or `ledger_index`. (See [Specifying a Ledger](#specifying-ledgers).) |
+| `ledger_index`     | String or Unsigned Integer                 | Use to look for transactions from a single ledger only. This field is optional and unnecessary if you use `ledger_index_min`, `ledger_index_max`, or `ledger_hash`. (See [Specifying a Ledger](#specifying-ledgers).) |
+| `binary`           | Boolean                                    | _(Optional)_ Defaults to `false`. If set to `true`, returns transactions as hex strings instead of JSON. |
+| `forward`          | Boolean                                    | _(Optional)_ Defaults to `false`. If set to `true`, returns values indexed with the oldest ledger first. Otherwise, the results are indexed with the newest ledger first. (Each page of results may not be internally ordered, but the pages are overall ordered.) |
+| `limit`            | Integer                                    | _(Optional)_ Default varies. Limit the number of transactions to retrieve. The server is not required to honor this value. |
 | `marker`           | [(Not Specified)](#markers-and-pagination) | Value from a previous paginated response. Resume retrieving data where that response left off. This value is stable even if there is a change in the server's range of available ledgers. |
 
-[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/AccountTxSwitch.cpp "Source")
-There is also a deprecated legacy variation of the `account_tx` method. For that reason, we recommend *not using any of the following fields*: `offset`, `count`, `descending`, `ledger_max`, `ledger_min`.
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/AccountTxSwitch.cpp "Source")<br>
+
+**Note:** For WebSocket and JSON-RPC, there is also a deprecated legacy variation of the `account_tx` method. For that reason, Ripple recommends *not using any of the following fields*: `offset`, `count`, `descending`, `ledger_max`, and `ledger_min`. If you use any of these deprecated fields, the method does not support pagination.
```

Comment:
Updated note text to make it clear that if you use deprecated fields, pagination isn't supported. Updated note text to be WebSocket- and JSON-RPC-specific because the commandline still requires use of some deprecated fields.